playbook/spec/dummy/config/application.rb

```diff
@@ -1,19 +1,28 @@
-require_relative 'boot'
+# frozen_string_literal: true
 
+require_relative "boot"
+
+require "rails"
+# Pick the frameworks you want:
 require "action_controller/railtie"
 require "action_view/railtie"
+require "playbook"
 
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
-require "playbook"
 
 module Dummy
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.2
```

Comment:
Should this be `6.1` with the Rails upgrade?

Reply:
@thestephenmarshall this just loads the default configurations. Rails update process generates an initializer with all the configs that are not using the current rails defaults so we can enable one by one, and when we're ready we can update this setting. It's worth trying to enable everything in that file and see /cc @jasperfurniss